src/Shopsys/ShopBundle/Controller/Admin/AdministratorController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Controller\Admin;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Shopsys\FrameworkBundle\Controller\Admin\AdministratorController as BaseAdministratorController;
+use Shopsys\FrameworkBundle\Model\Administrator\Administrator;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+class AdministratorController extends BaseAdministratorController
+{
+    /**
+     * @Route("/administrator/edit/{id}", requirements={"id" = "\d+"})
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $id
+     */
+    public function editAction(Request $request, $id)
+    {
+        /** @var \Shopsys\ShopBundle\Model\Administrator\Administrator $administrator */
+        $administrator = $this->administratorFacade->getById($id);
+
+        if ($administrator->isDefaultAdmin()) {
+            $message = 'Default admin cannot be edited.';
+            throw new AccessDeniedException($message);
```

Comment:
I test this in production mode and it showed me the 404 page in the frontend look. 
It's confusing and I'm not sure if it's intended behavior.

Reply:
Currently we have only 404 and 500 error pages. I agree with you, that it is not ideal, but in this case (when there is no direct link) I think it is ok.